Apps/Sandcastle/gallery/Geometry Height Reference.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
```

Comment:
My comments in https://github.com/AnalyticalGraphicsInc/cesium/pull/6434#issuecomment-390754541 still apply here. I would also like to see some inline comments to explain the feature.  If I'm just trying to learn Cesium's capabilities from Sandcastle examples, there's very little context here.